Add use case for lowest memory consumption [8912] (#177)
eProsima/Fast-RTPS-docs: docs/fastdds/use_cases/use_cases.rst

  this tutorial is an explanation of how to take full advantage of *Fast DDS* wide set of capabilities in a ROS 2
  project.

> + :ref:`use-case-reduceMemory`.

Reviewer: Update the link. The documentation doesn't pass the test right now.

Reviewer: This is not resolved.

>   For use cases where available memory is scarce, *Fast DDS*  can configured to reduce memory footprint

Reviewer: ```suggestion
  For use cases where available memory is scarce, *Fast DDS*  can be configured to reduce memory footprint
```

>   For use cases where available memory is scarce, *Fast DDS*  can configured to reduce memory footprint

Reviewer: Memory doesn't have to be scarce, you simply have memory use constraints.

```suggestion
  For use cases with memory consumption constraints, *Fast DDS*  can configured to reduce memory footprint
```

Reviewer: This is not resolve either.

>   to a minimum by adjusting different  QoS policies.

Reviewer: ```suggestion
  to a minimum by adjusting different QoS policies.
```

>   For use cases were available memory is scarce this example presents configuration options to reduce memory footprint
>   to a minimum by tweaking the :ref:`dds_layer_core_policy`.

Reviewer 2: Qos Policies should be a link
```suggestion
  For use cases where available memory is scarce, *Fast DDS*  can configured to reduce memory footprint
  to a minimum by adjusting different  QoS policies.
```
